plugins/auth-auth0/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@amplication/plugin-auth-auth0",
+  "version": "1.0.0",
+  "description": "Auth0 plugin for Amplication",
+  "main": "dist/index.js",
+  "scripts": {
+    "prepublishOnly": "npm run build",
+    "dev": "webpack --watch",
+    "build": "webpack",
+    "prebuild": "rimraf dist",
+    "format:write": "prettier --write \"**/*.{js,ts}\"",
+    "lint:fix": "eslint './src/**/*.{ts, tsx}' --fix --c .eslintrc",
+    "lint": "eslint './src/**/*.{ts, tsx}' --c .eslintrc",
+    "test": "jest"
+  },
+  "author": "Ashish Padhy <100484401+Shurtu-gal@users.noreply.github.com>",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@amplication/code-gen-types": "file:../../../amplication/dist/packages/amplication-code-gen-types/",
+    "@amplication/code-gen-utils": "^0.0.5",
+    "@babel/parser": "^7.20.7",
+    "@babel/types": "^7.20.7",
+    "@types/jest": "^29.5.3",
+    "@types/lodash": "^4.14.197",
+    "@typescript-eslint/eslint-plugin": "^6.2.1",
+    "@typescript-eslint/parser": "^6.2.1",
+    "copy-webpack-plugin": "^11.0.0",
+    "eslint": "^8.46.0",
+    "jest-mock-extended": "^3.0.1",
+    "lodash": "^4.17.21",
+    "prettier": "^2.8.7",
+    "rimraf": "^4.4.1",
+    "ts-jest": "^29.1.1",
+    "ts-loader": "^9.4.2",
+    "ts-node": "^10.9.1",
+    "tsconfig-paths-webpack-plugin": "^4.1.0",
+    "typescript": "^4.9.3",
+    "webpack": "^5.76.0",
+    "webpack-cli": "^5.0.1",
+    "webpack-node-externals": "^3.0.0"
```

Comment:
Please align the dependencies with the latest master dependencies of the plugin-template